tests/test_wave_07.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from swap_meet.vendor import Vendor
+from swap_meet.item import Item
+
```

Comment:
Great job creating these tests for the newest item. Another test to consider is handling ties for the newest item.